### ISSUE TITLE
Cannot read property 'ipns' of undefined

### ISSUE THREAD

**Issue by HuangJie98** [dealer-platform/dapp#1]
When accessing "user profile (mypage)", the“UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'ipns' of undefined“

**Comment by sigma67** [comment]
Did you initialize the database tables?

**Comment by HuangJie98** [comment]
Initialize the corresponding user on the EOS Studio?If so, I did.

**Comment by sigma67** [comment]
No, you need to also call reporting::init to initialize the contract with test data

https://github.com/dealer-platform/smartContract/blob/d89274bd27a9d48007b9a81e1ac3212d9b74c1c6/src/modules/initialize.cpp#L29-L31

**Comment by HuangJie98** [comment]
Like this?
![S42A 0FF$6TFSY`BRGB(13D](https://user-images.githubusercontent.com/60546464/123768117-a107d200-d8fa-11eb-91d2-bba0ba95ede8.png)

Still the Same.

**Comment by sigma67** [comment]
You also need to set up a config.json based on config.json.example

https://github.com/dealer-platform/dapp/blob/master/config.json.example

**Comment by HuangJie98** [comment]
Besides modifying
[https://github.com/dealer-platform/dapp/blob/master/config.json.example#L4-L5](url),
[https://github.com/dealer-platform/dapp/blob/master/config.json.example#L4-L5](url),
[https://github.com/dealer-platform/dapp/blob/master/config.json.example#L15-L16](url).
Do I need to modify anything else?

**Comment by sigma67** [comment]
No, lgtm

**Comment by HuangJie98** [comment]
But I've changed it before, and there are still problems.
